x-pack/plugins/alerting/server/rules_client/rules_client.ts

```diff
@@ -180,6 +211,63 @@ export interface FindOptions extends IndexType {
   filter?: string;
 }
 
+type BulkEditFields = keyof Pick<Rule, 'actions' | 'schedule' | 'tags' | 'throttle' | 'notifyWhen'>;
```

Comment:
Would this be correct? given the comment below.

```suggestion
type BulkEditFields = keyof Pick<Rule, 'actions' | 'tags'>;
```

Reply:
Yeah, would be. I've added this change.

Reply:
Looks good, thanks!